src/ocm/components/newFeatureSupportLevels/newFeatureStateUtils.ts

```diff
@@ -154,7 +154,3 @@ export const getNewFeatureDisabledReason = (
     }
   }
 };
-
-export const isFeatureSupported = (supportLevel: SupportLevel | undefined) => {
```

Comment:
I just realised that, if this function had been called initially everywhere were we were checking if a feature was `unsupported`, this PR would have been a one liner.

Also, looking at it, it appears like the new function `isFeatureNotUnsupportedNotUnavailable` makes more sense to move it to this file `newFeatureStateUtils`, rather that on the current `cpuArchitecture`